presentation/components/src/presentation-components/tree/controlled/WithFilteringSupport.tsx

```diff
@@ -82,9 +82,9 @@ export function DEPRECATED_controlledTreeWithFilteringSupport<P extends Controll
     return (
       <div className="filteredTree">
         <TreeComponent
-          nodeLoader={nodeLoader}
           nodeHighlightingProps={nodeHighlightingProps}
           {...strippedProps as CombinedProps}
+          nodeLoader={nodeLoader}

```

Comment:
@grigasp  Is it safe to change the ordering for these?  tsc was pointing out that `CombinedProps` still contains non-nullable `nodeLoader` and `visibleNodes` props...

Reply:
see also: presentation/components/src/presentation-components/tree/controlled/WithVisibleNodes.tsx

Reply:
I think it's fine. @saskliutas what's your opinion?

Reply:
It should be fine.